tests/routes/patch_booking_by_token_v2_test.py

```diff
@@ -0,0 +1,497 @@
+from urllib.parse import urlencode
+
+
+from models import PcObject, EventType, ThingType, Deposit, Booking, User
+from tests.conftest import clean_database, TestClient
+from tests.test_utils import create_deposit, create_venue, create_offerer, \
+    create_user, create_booking, create_offer_with_event_product, \
+    create_event_occurrence, create_stock_from_event_occurrence, create_user_offerer
+from tests.test_utils import create_api_key, create_stock_with_event_offer
+from utils.human_ids import humanize
+from utils.token import random_token
+
+API_KEY_VALUE = random_token(64)
+
+class Patch:
+    class Returns204:
+        class WithApiKeyAuthTest:
+            @clean_database
+            def when_api_key_is_provided_and_rights_and_regular_offer(self, app):
+                # Given
+                user = create_user()
+                offerer = create_offerer()
+                venue = create_venue(offerer)
+                stock = create_stock_with_event_offer(offerer, venue, price=0)
+                booking = create_booking(user, stock, venue=venue)
+                PcObject.save(booking, offerer)
+
+                offererApiKey = create_api_key(offerer, API_KEY_VALUE)
```

Comment:
Mon linter râle un peu sur le camelCase ^^
```suggestion
                offerer_api_key = create_api_key(offerer, API_KEY_VALUE)
```

Reply:
👍 
PyCharm tolère :(